integration/docker.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"context"
+	"testing"
+
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/container-utils/testutils"
+)
+
+// DockerContainer implements Operation for docker containers
+type DockerContainer struct {
+	Name    string
+	Cmd     string
+	Options []dockerOption
+
+	Cleanup      bool
+	Started      bool
```

Comment:
It doesn't need to be exposed, isn't it?

```suggestion
	started      bool
```